UKHomeOffice/cerberus-service
# COp-4489: Wrap structure in keycloak auth

## Description
Wrap structure in keycloak auth.

## To Test
- Pull the branch locally;
- Install dependencies;
- Run the project;
- Go to `http://localhost:8080` and you should be redirected to keycloak login page;
- Login with O365, and you should see the menu;

## Developer Checklist

\* Required
